src/schema-parser/asyncapi-schema-parser.ts

```diff
@@ -8,12 +19,73 @@ export function AsyncAPISchemaParser(): SchemaParser {
   }
 }
 
-function validate() {
+async function validate(input: ValidateSchemaInput<unknown, unknown>): Promise<SchemaValidateResult[]> {
+  const version = input.asyncapi.semver.version
+  const validator = findSchemaValidator(version);
+
+  let result: SchemaValidateResult[] = []
+  const valid = validator(input.data);
+  if (!valid && validator.errors) {
+    result = ajvToSpectralResult(validator.errors, input.path);
+  }
+
+  return result;
+}
+
+function ajvToSpectralResult(errors: ErrorObject[], parentPath: Array<string | number>): SchemaValidateResult[] {
+  if (parentPath === undefined) {
+    parentPath = [];
+  }
+
+  return errors.map(error => {
+    const errorPath = error.instancePath.replace(/^\//, '').split('/'); // TODO: Instance Path or Schema Path?
```

Comment:
Instance path, because we should inform about problems related on inputs, not on JSON Schema.